lib/dojos.js

```diff
@@ -323,6 +323,7 @@ exports.register = function (server, options, next) {
             alpha3: joiValidator.alpha3().required(),
             address1: Joi.string().required(),
             coordinates: Joi.string().required(),
+            attendee: Joi.string().required(),
```

Comment:
I agree - it could be more clear. I've refactored to make it more understandable and to use input type = number also.
Thanks! :)